packages/FieldTypes/src/components/Container/Container.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as sc from "./Container.styles";
```

Comment:
Nitpick: I believe we usually want to put the styles import last

Reply:
the only linter requirement that we have is to put imports from node_modules before any local import, I would think is Niptick but I can change it :)